apps/shell/tash_security.c

```diff
@@ -0,0 +1,77 @@
+/****************************************************************************
+ *
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+/// @file   tash_security.c
+/// @brief  functions to check secure tash command
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+#include <mbedtls/sha256.h>
+#include "tash_internal.h"
+
+void tash_check_security(int fd)
+{
+	char *line_buff;
+	int i;
+	int nbytes;
+	char chr[1];
+	unsigned char temp[2];
+
+	unsigned char sha256[64];
+	unsigned char result[64];
+	unsigned char sha256_hex[64];
+	int len = 0;
+	const char passwd_prompt[] = "PASSWORD>>";
+	int count = 0;
```

Comment:
Is this ok for TASH stack?

Reply:
TASH stack is 4K by default, so it looks okay.